datastore/kubernetes.go

```diff
@@ -434,7 +434,15 @@ func NewPVManifestForVolume(v *longhorn.Volume, pvName, storageClassName, fsType
 		"staleReplicaTimeout": strconv.Itoa(v.Spec.StaleReplicaTimeout),
 	}
 
-	return NewPVManifest(v.Spec.Size, pvName, v.Name, storageClassName, fsType, volAttributes)
+	// by default we create rwo manifests
```

Comment:
Why not directly modify the function `NewPVManifest`? Same below

Reply:
updated to pass the access mode specifically when creating the manifest